samples/openthread/coprocessor/sample.yaml

```diff
@@ -1,8 +1,8 @@
 sample:
-  name: OpenThread NCP sample
-  description: Test OpenThread NCP
+  name: OpenThread Co-Processor sample
+  description: Test OpenThread Co-Processor
 tests:
-  samples.openthread.ncp:
+  samples.openthread.coprocessor:
     build_only: true
     build_on_all: true
     platform_allow: nrf52840dk_nrf52840 nrf52833dk_nrf52833
```

Comment:
If you have some time, please provide also the overlay files for automatic building. I was told it is possible to specify overlay files applicable for each platform and the CI will try to rebuild all possible configurations automatically.

If you don't want to it is ok, to ignore this comment as this is not in the scope of your task.

Reply:
Who told you that? It would be much help if I had a contact to the person who know a little bit at least about the mention feature.

Reply:
I think it was @carlescufi on one of the release sync meetings.